File: examples/setGPIO.c
#include <stdio.h>
#include <string.h>
#include <time.h>

#include <pi-gpio.h>

#define PinA 17
#define PinB 18

int main() {
  setup();

    printf("Set GPIO%d INPUT GPIO%d OUTPUT,HIGH\n", PinA, PinB);

  setup_gpio(PinA, INPUT, 0);  // Set GPIO 17 INPUT no PU
  setup_gpio(PinB, OUTPUT, 0); // Set GPIO 18 OUTPUT
  output_gpio(PinB, 1);        // Set GPIO HIGH

  cleanup();

  return 0;
}

File: examples/setGPIO.py
#! /usr/bin/env python3
# 2022-11-25

import pi_gpio as GPIO

GPIO.setup()

PinA = 17
PinB = 18

print("Set GPIO: {} INPUT; GPIO: {} OUTPUT,HIGH".format(PinA, PinB))

GPIO.setup_gpio(PinA, 0, 0) # Set PinA INPUT no PU
GPIO.setup_gpio(PinB, 1, 0); # Set GPIO 18 OUTPUT
GPIO.output_gpio(PinB, 1) # Set PinB HIGH

GPIO.cleanup()
